task/idle/src/main.rs

```diff
@@ -31,9 +28,12 @@ fn main() -> ! {
             // which could wake some higher-priority task.
             cortex_m::asm::wfi();
         }
+
         // RISC-V has wfi, but unfortunately it is an illegal instruction if
-        // called from User mode, so instead we spin on a timer call.
-        #[cfg(target_arch = "riscv32")]
-        while sys_get_timer().now > 0 {}
+        // called from User mode.
+        #[cfg(any(target_arch = "riscv32", target_arch = "riscv64"))]
+        unsafe {
+            riscv::asm::nop();
+        } // Heed the warning above.
```

Comment:
What is this referring to?

Reply:
This is the warning about using an empty loop - which the task author suggested was UB (at least at the point they were writing it).